memory-db/Cargo.toml

```diff
@@ -11,6 +11,7 @@ heapsize = { version = "0.4", optional = true }
 parity-util-mem = { version = "0.2", default-features = false }
 hash-db = { path = "../hash-db", default-features = false, version = "0.15.2"}
 hashbrown = { version = "0.6.3", default-features = false, features = [ "ahash" ] }
+ahash = "0.2.18"
```

Comment:
There's a compilation error with `0.2.17`, which is permitted by the `0.2.11` constraint in `hashbrown`.